modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/management_bar/components/filters/AutocompleteFilter.js

```diff
@@ -14,27 +14,35 @@
 
 import ClayAutocomplete from '@clayui/autocomplete';
 import ClayButton from '@clayui/button';
-import {ClayCheckbox, ClayRadio} from '@clayui/form';
+import ClayDropDown from '@clayui/drop-down';
+import {ClayCheckbox, ClayRadio, ClayToggle} from '@clayui/form';
 import ClayLabel from '@clayui/label';
 import ClayLoadingIndicator from '@clayui/loading-indicator';
 import {useIsMounted} from 'frontend-js-react-web';
 import {fetch} from 'frontend-js-web';
 import PropTypes from 'prop-types';
 import React, {useCallback, useEffect, useRef, useState} from 'react';
 
-import {fetchParams, getValueFromItem} from '../../../utilities/index';
+import {
+	getAcceptLanguageHeaderParam,
+	getValueFromItem,
+	isValuesArrayChanged,
+} from '../../../utilities/index';
 import {logError} from '../../../utilities/logError';
 
 const DEFAULT_PAGE_SIZE = 10;
 
 function fetchData(apiURL, searchParam, currentPage = 1) {
 	return fetch(
-		`${apiURL}?page=${currentPage}&pageSize=${DEFAULT_PAGE_SIZE}${
+		`${apiURL}${
```

Comment:
Could we extract the url composition so it can be a little more readable?

Reply:
We could probably use `new URL()` and `url.searchParams` to make the url construction a bit more clear here.